web/src/redux/selectors/apd.selectors.js

```diff
@@ -72,19 +72,58 @@ export const selectPreviousHITHIEActivities = createSelector(
     )
 );
 
-export const selectPreviousMMISActivities = createSelector(
+export const selectPreviousMMISActivitiesHITECH = createSelector(
   [selectApdData],
   ({ previousActivities }) =>
     Object.entries(previousActivities.actualExpenditures).reduce(
       (o, [year, expenses]) => ({
         ...o,
-        [year]: expenses.mmis
+        [year]: expenses
```

Comment:
Wouldn't this still be referenced the same for hitech? As in, don't we still need the `.mmis`?

Or it looks like you're getting all the expenses with this selector. So maybe taking MMIS out of the name?

Reply:
For MMIS, the mmis object was replaced with ddi and m&o. So I had to remove the .mmis
For HITECH, there is a mmis and hithie object, so I'm using this selector to pass in both of those and just reference them in the file